<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>图片名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="headLaebl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>54</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>头像</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>54</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeTextLabel">
   <property name="geometry">
    <rect>
     <x>1133</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>已标注时间:</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>10</y>
     <width>54</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="labelBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>241</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>元素标签</string>
   </property>
   <widget class="QPushButton" name="plusLabelButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subLabelButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QListView" name="labelListView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>241</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="finishBox">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>80</y>
     <width>241</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>已标注信息</string>
   </property>
   <widget class="QPushButton" name="delButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QListView" name="finishListView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>241</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="toolsBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>工具栏</string>
   </property>
   <widget class="QPushButton" name="suggestButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>推荐标签</string>
    </property>
   </widget>
   <widget class="QPushButton" name="autoButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>自动标注</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearLabelButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清空标签</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redoButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>重做</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="menuBox" native="true">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>510</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QPushButton" name="skipButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>跳过</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="xyLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X:  Y:  </string>
   </property>
  </widget>
  <widget class="QWidget" name="imgWidget" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>741</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="recvLabel">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>已接收：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
